lib/datadog/appsec.rb

```diff
@@ -8,6 +8,16 @@ module Datadog
   module AppSec
     include Configuration
 
+    class << self
+      def components
+        Datadog.send(:components)
+      end
```

Comment:
I will move it to the private block. Also, to be fair, nothing within the appsec module is marked as public

Reply:
> Also, to be fair, nothing within the appsec module is marked as public

Right... You are right that public methods are "supposed to" be marked with `@public_api`, although I would argue that we should be extra cautious about what we accidentally expose in the top-level product modules, because of expectations set in other components -- e.g. everything "public" at the Ruby level in `Datadog::Tracing`, `Datadog::Profiling` is supposed to be a `@public_api`, so it may lead customers to assume the same was true for `Datadog::AppSec`.